sdk/src/main/java/io/radar/sdk/RadarBeaconManager.kt

```diff
@@ -300,6 +300,10 @@ internal class RadarBeaconManager(
             return
         }
 
+        if (!this::adapter.isInitialized) {
+            adapter = BluetoothAdapter.getDefaultAdapter()
```

Comment:
It just gets a handle to the default system Bluetooth adapter. Doesn't initialize it per se: https://developer.android.com/reference/android/bluetooth/BluetoothAdapter#getDefaultAdapter()

It can return `null` if Bluetooth is unavailable. In most cases `context.packageManager.hasSystemFeature(PackageManager.FEATURE_BLUETOOTH)` will return `false` if this is the case, but I think this can very from manufacturer to manufacturer, which is why we're seeing this crash.